grovedb/Cargo.toml

```diff
@@ -34,6 +34,29 @@ name = "insertion_benchmark"
 harness = false
 
 [features]
-default = ["full"]
-full = ["merk/full"]
-verify = ["merk/verify"]
\ No newline at end of file
+default = ["verify"]
```

Comment:
I'm wondering what will it be without any features enabled lol

It's the reason why `#[cfg(any(feature = "full", feature = "verify"))]` is required, could be just unconditional compilation, like "basic GroveDB package"

Reply:
Doing this in a new pr